lib/steps/setupRedirects.js

```diff
@@ -45,7 +45,21 @@ const setupRedirects = () => {
     // One route may map to multiple Netlify routes: e.g., catch-all pages
     // require two Netlify routes in the _redirects file
     getNetlifyRoutes(route).map((netlifyRoute) => {
-      const redirect = `${netlifyRoute}  ${nextRedirect.target}  200`;
+      const {
+        conditions = [],
+        force = false,
+        statusCode = "200",
+        target,
+      } = nextRedirect;
+      const redirectPieces = [
+        netlifyRoute,
+        target,
+        `${statusCode}${force ? "!" : ""}`,
+      ];
```

Comment:
```suggestion
        conditions.join("  ")
      ];
```

Reply:
didnt include this from the first review because it causes extra spaces at the end, empty array will put an empty string here and the main array's join will count it